helms/legion/templates/edge/edge-server.yaml

```diff
@@ -4,51 +4,65 @@ kind: Deployment
 metadata:
   name: "{{ .Release.Name }}-edge"
   labels:
-    heritage: {{ .Release.Service | quote }}
-    release: {{ .Release.Name | quote }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
-    component: "{{ .Release.Name }}-edge"
+    component: "legion-edge"
+    {{- include "legion.helm-labels" . | nindent 4 }}
 spec:
   replicas: {{ .Values.edge.replicas }}
   selector:
     matchLabels:
-      component: "{{ .Release.Name }}-edge"
+      component: "legion-edge"
+      {{- include "legion.helm-labels-for-search" . | nindent 6 }}
   template:
     metadata:
       labels:
-        component: "{{ .Release.Name }}-edge"
+        component: "legion-edge"
+        {{- include "legion.helm-labels" . | nindent 8 }}
     spec:
       serviceAccountName: "{{ .Release.Name }}-edge"
+      {{- if .Values.feedback.enabled }}
       volumes:
       - name: edge-scripts
         configMap:
           name: "{{ .Release.Name }}-edge-scripts"
+      {{- end }}
       containers:
       - name: nginx
-        image: "{{ .Values.edge.image.repository }}:{{ .Values.edge.image.tag }}"
+        image: "{{ include "legion.image-name" (dict "root" . "service" .Values.edge "tpl" "%sk8s-edge:%s") }}"
         resources:
           requests:
```

Comment:
We can simplify it using the following template:
```yaml
      containers:
      - name: edi
        image: "{{ .Values.edi.image }}"
        imagePullPolicy: Always
{{- if .Values.edi.resources }}
        resources:
{{ toYaml .Values.edi.resources | indent 10 }}
{{- end }}
        env:
        - name: DEPLOYMENT
          value: "{{ .Release.Name }}"
```

Reply:
done